src/Kestrel.Core/Kestrel.Core.csproj

```diff
@@ -1,10 +1,10 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <AssemblyName>Microsoft.AspNetCore.Server.Kestrel.Core</AssemblyName>
     <RootNamespace>Microsoft.AspNetCore.Server.Kestrel.Core</RootNamespace>
     <Description>Core components of ASP.NET Core Kestrel cross-platform web server.</Description>
-    <TargetFrameworks>netstandard2.0;netcoreapp2.1</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netcoreapp2.2</TargetFrameworks>
```

Comment:
Also https://github.com/aspnet/KestrelHttpServer/blob/8b1fbad10e12e7ddab98ae7a17891eca6e6bcfd9/src/Kestrel.Core/Internal/HttpsConnectionAdapter.cs#L129

Reply:
Actually, this one likely needs to stay at 2.1. It was only added for new API. There's no reason to move it higher right now.